src/server/routes/agent.rs

```diff
@@ -109,8 +109,7 @@ fn endpoint_next_experiment(data: Arc<Data>, auth: AuthDetails) -> Fallible<Resp
             }
         }
 
-        ex.remove_completed_crates(&data.db)?;
```

Comment:
Why did you remove this? Removing completed crates from the list returned to the agent allows us to resume stopped experiments without starting from scratch.

Reply:
I removed `remove_completed_crates` because it just change the value of the `crates` field, that is not present anymore, but then I forgot to filter crates in get_crates().  
Working to fix this